src/niamoto/cli/utils/console.py

```
@@ -1,32 +1,329 @@
 
 
-def print_success(message: str) -> None:
+# Detect if we're on Windows and if emoji support might be limited
+def _should_use_emojis() -> bool:
+    """Determine if emojis should be used based on platform and terminal capabilities."""
+    # Check environment variable override
+    emoji_env = os.getenv("NIAMOTO_USE_EMOJIS", "").lower()
+    if emoji_env in ("1", "true", "yes", "on"):
+        return True
+    elif emoji_env in ("0", "false", "no", "off"):
+        return False
```

Pylint (3.3.7)

[refactor] Unnecessary "elif" after "return", remove the leading "el" from "elif"

(R1705)